content/guides/farmlands-pos-integration.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Farmlands POS Integration Guide
+img: /farmlands-pos-integration-cover.jpg
```

Comment:
@MeganSteenkamp I think it's good that you used the front-matter to declare the image path, @AndyClifford will need this path to display this guide as a small card on the partner page 👍 

Is there a reason why it doesn't need to be the full path (include `/public`)?

Reply:
Is it possible to use the assets directory here? `~/assets/farmlands/pos-integration-cover.jpg`

Reply:
@tom-blake @AndyClifford I have tried the assets directory but it didn't work 🤔. By default files in the `/public` directory are served at the root which is why I didn't need to specify the folder.

I see Nuxt has an image module if we did want to optimize images: https://v1.image.nuxtjs.org/get-started/

Reply:
@MeganSteenkamp Yeah it's not working for me either. [Found this](https://github.com/nuxt/content/issues/106#issuecomment-1215530462) comment that suggests copying the files to `public` or using the module you mentioned